openSUSE/open-build-service
# Move package edit action close to the target

Following the conventions used for editing the details of a kiwi image, I've:

- Moved the "Edit Package" button close to the edited details.
- Reimplemented the Cancel button without ajax.
- Fixed some markup issues when saving the modified details

**Before**
![image](https://user-images.githubusercontent.com/2650/111345396-88421c00-867d-11eb-9862-5c9dc80579d3.png)

**After**
![image](https://user-images.githubusercontent.com/2650/111345544-adcf2580-867d-11eb-9d82-f2ecf58871dd.png)


Supersedes #10874

Fixes #10510

How to review it:
- Go to a package overview
- Click on the "Edit" button below the package details. The package details get replaced by a form to edit those details.
- Click on Cancel to get the package details back.

_Replace this line with a description of your changes. Please follow the suggestions in the comment below._

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:


[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature
